docs/installation.md

```diff
@@ -39,15 +39,20 @@ two examples.
 Bind-mount your `prometheus.yml` from the host by running:
 
 ```bash
-docker run -p 9090:9090 -v /tmp/prometheus.yml:/etc/prometheus/prometheus.yml \
-       prom/prometheus
+docker run \
+    -p 9090:9090 \
+    -v /tmp/prometheus.yml:/etc/prometheus/prometheus.yml \
+    prom/prometheus
 ```
 
 Or use an additional volume for the config:
 
 ```bash
-docker run -p 9090:9090 -v /prometheus-data \
-       prom/prometheus --config.file=/prometheus-data/prometheus.yml
+docker run \
+    -p 9090:9090 \
+    -v /path/to/data:/prometheus-data \
+    prom/prometheus \
+    --config.file=/prometheus-data/prometheus.yml
```

Comment:
Let's remove this, and provide an example for volume mapping the config. Something like `-v /path/to/config:/etc/prometheus`.

Reply:
Sounds good! Updated with your suggestion.